websites/D/discord.style/dist/metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "author": {
+    "name": "bris",
+    "id": "656165298818514946"
+  },
+  "service": "discord.style",
+  "description": {
+    "en": "Discord.style is a website built to help make a discord server easier."
```

```suggestion
    "en": "Online library for finding and sharing free Discord Templates. Upload your own Discord Template and collect votes!"
```
(Their official description)